tools/cli/src/webpack/config.ts

```diff
@@ -76,6 +76,10 @@ export const getPublicPath = (buildFlags: BuildFlags) => {
     return publicPath;
   }
 
+  if (buildFlags.distribution === 'mobile') {
+    return '/mobile/';
```

Comment:
this will make affine itself fail. it should be joint with CDN prefix

Reply:
Reasonable, I will change here.